api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/AutowiringSpringBeanJobFactory.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.dhsc.htbhf.claimant.scheduler;
+
+import org.quartz.spi.TriggerFiredBundle;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.scheduling.quartz.SpringBeanJobFactory;
+
+/**
+ * {@link org.quartz.spi.JobFactory} implementation that is spring aware, allowing {@link org.quartz.Job} instances
+ * to autowire spring bean dependencies.
+ * {@see <a href="https://dzone.com/articles/spring-and-quartz-integration-that-works-together/> here </a>} for a more complex example
+ * which gives the job access to {@link org.quartz.JobDataMap}s and the {@link org.quartz.SchedulerContext}. (not need for our job)
+ */
+public class AutowiringSpringBeanJobFactory extends SpringBeanJobFactory implements ApplicationContextAware {
```

Comment:
Its weird that GitHub isn't colour coding this class, is there a stray bit of commenting out that means this is commented out? Can't see one but thought I'd mention it.

Reply:
I can't see any issues in my ide, and the build passes fine for me.

Reply:
Fixed javadoc error